index.bs

```diff
@@ -5080,6 +5157,58 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 ### Events ### {#module-script-events}
 
+#### The script.message Event #### {#event-script-message}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        script.Message = {
+         method: "script.message",
+         params: script.MessageParameters
+       }
+
+       script.MessageParameters = {
+         channel: script.Channel,
+         data: RemoteValue
+         realm: script.Realm
```

Comment:
As a client, in order to understand from which BrowsingContext the event came, I have to track the map of realm-BrowsingContext by subscribing to and handling `script.realmCreated` + `script.realmDestroyed`. 

Consider adding field `context: browsingContext.BrowsingContext / null`. Nullable to support workers in future.